### ISSUE TITLE
Autofocus issue title field and fix tab index in issue editing screen

### ISSUE THREAD

**Issue by trejder** [gogits/gogs#1422]
Can we have `autofocus="autofocus"` added to `<input name="title">` field in Gogs, so clicking `New issue` would place cursor in that field by default? This would allow user to start typing issue title right away, without need for an additional click to title field.

Exactly as it works here, on GitHub. It speeds up work extremely, if you have to enter a lot of issues at once (i.e. upon project manual migration).

When editing existing issue, issue body field should be auto-focused, instead of issue title, as in most cases you're changing / fixing issue body, not title.

In addition, you should consider changing tab-index of objects in Issue edit screen (if this is possible) to require presing `Tab` button only once to go from editing issue title to writing issue body. Again, exactly as this happens in GitHub issues. Other elements in-between (assigned / milestone dropdowns, tabs) shouldn't be "reachable"  by `Tab` button at all or should have tab index set so would be active _after_ issue body, not _before_ it. Thank you.

**Comment by Unknwon** [comment]
- [x] New issue
- [ ] Edit issue
- [x] New milestone
- [x] Edit milestone

**Comment by Unknwon** [comment]
It is now possible to edit issue and comments! https://try.gogs.io/unknwon/gogs/issues
